src/main/java/com/mock/preparation/ds/Unique.java

```diff
@@ -0,0 +1,33 @@
+package mock.preparation.ds;
+
+public class Unique {
+
+    /*
+    Find the unique element in an array
+     */
+
+    public static void unique(int[] a) {
+
+        for (int i = 0; i < a.length; i++) {
+            boolean isunique = false;
+            for (int j = 0; j < i; j++) {
+                if (a[i] == a[j]) {
+                    isunique = true;
+                    break;
+                }
+            }
+            if (!isunique) {
+                System.out.println(" " + a[i]);
+            }
+        }
+    }
+
+    public static void main(String[] args) {
+        int arr[] = {6, 6, 7, 2, 3, 1, 4, 45, 21, 3, 4};
+        // Arrays.sort(arr);
+        unique(arr);
+    }
+}
```

Comment:
is this correct as discuss yesterday is your output is correct again. don't commit as just shake of commit. it should b correct as well. you will be refering all this after some time.